Documentation/security/policy/language.rst

```diff
@@ -503,6 +504,22 @@ but not CIDR prefix ``10.96.0.0/12``
 
         .. literalinclude:: ../../../examples/policies/l3/cidr/cidr.json
 
+.. _cidr_select_nodes:
+
+Selecting nodes with CIDR / ipBlock
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../../beta.rst
+
+By default, CIDR-based selectors do not match in-cluster entities (pods or nodes).
+Optionally, the policy engine can be directed to select nodes via CIDR / ipBlock.
```

Comment:
No Latin abbreviations
```suggestion
Optionally, the policy engine can be directed to select nodes by CIDR / ipBlock.
```

Reply:
But how else am I going to show off my overpriced education :stuck_out_tongue_closed_eyes: